lib/Fetcher/CodeFetcher/Download.php

```diff
@@ -31,7 +31,7 @@ public function setup() {
       drush_set_option($name, $value);
     }
     if ($this->site['verbose']) {
-      $command = 'drush dl ' . implode(' ', $commandline_args) . ' --verbose';
+      $command = 'drush dl ' . implode(' ', $commandline_args) . '--cache=0' . ' --verbose';
```

Comment:
Why concatenate `--cache` as a separate string, that should just be one string concatenated here for readability.